frontend/shared/src/utils/__tests__/date.utils.test.ts

```diff
@@ -0,0 +1,27 @@
+import { diffMonths, parseDate } from '../date.utils';
+
+describe('dates', () => {
+  describe('diffMonths', () => {
+    it('should be 0 if dates are equal', () => {
+      const diffMonthsResult = diffMonths(
+        parseDate('1.12.2020') || 0,
+        parseDate('1.12.2020') || 0
+      );
+      expect(diffMonthsResult).toBe(0);
+    });
+    it('should be 12.13 months between 1.01.2021 and 31.12.2021', () => {
+      const diffMonthsResult = diffMonths(
+        parseDate('31.12.2021') || 0,
```

Comment:
Looks good! Just wondering if ` || 0 ` would be inside the diffMonths, so it would accept empty value and default it inside the method